src/status_im2/contexts/quo_preview/wallet/account_overview.cljs

```diff
@@ -0,0 +1,104 @@
+(ns status-im2.contexts.quo-preview.wallet.account-overview
+  (:require [quo2.core :as quo2]
```

Comment:
Avoid aliasing to `quo2` unless strictly necessary. This will save us the trouble of changing lots of references from `quo2` to `quo` in the future (once we delete the old lib).